app/services/bulk_upload/sales/log_creator.rb

```diff
@@ -25,6 +25,7 @@ def call
         Sentry.capture_exception(e)
       end
     end
+    @bulk_upload.update!(noint_fix_status: BulkUpload.noint_fix_statuses[:not_needed])
```

Comment:
Is this the right place to add this? Would it make sense upon bulk_upload creation?

Reply:
You can't do it on bulk upload creation because that happens from an up instance, so something could be created with updated app code but still processed with the old parser by sidekiq (if it was uploaded at exactly the wrong moment).

Although actually I think maybe it needs to happen at the start of this function rather than the end, in case of running the task at exactly the wrong moment.

Reply:
> Wouldn't `@bulk_upload.update!(noint_fix_status: "not_needed")` work/ mean the same thing?

Yes - I was generally avoiding typing the string because I inevitably end up making typos, and referencing the enum itself forces it to be right.

Reply:
That's fair in terms of timing
I think a typo would cause an error, so shouldn't be too bad to use here (given there are tests)
<img width="703" alt="image" src="https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/assets/54268893/e23f65cc-5878-43df-88bd-bde54b75977a">